### ISSUE TITLE
Error while using wxplot3d

### ISSUE THREAD

**Issue by leafina2895** [wxMaxima-developers/wxmaxima#1960]
I do not know, if this is the correct place to ask, but I give it a go nonetheless:

OS: Arch Linux x86_64 6.10.10-arch1-1
maxima Version: 5.47.0
wxmaxima Version: 24.08.0_DevelopmentSnapshot
gnuplot Version: 6.0 patchlevel 1
(using Wayland)

When using wxplot3d I get the following error and a blank color:
```
set pm3d hidden3d 100 border lw 0.5 lt rgb "#000000"
                  ^
"/tmp/maxout19174.gnuplot" line 10: invalid pm3d option
``` 

Plotting it with 'plot3d' works, even though an error is still shown.

Researching on Sourceforge (maxima) I found that someone encountered the same bug, but it has been said to have been patched. If anyone has an idea what might be the error, I'd be happy.

**Comment by daute** [comment]
Yes, I remember that, the bug report on Sourceforge was made by me.
(For reference: https://sourceforge.net/p/maxima/bugs/4297/). A problem with Maxima and Gnuplot 6.

The issue was fixed by  Gunter Königsmann (thanks!) - on 2024-05-11. And you are using Gnuplot 6 and Maxima 5.47.0 (released in June 2023). No Maxima release was made after the solution from Gunter.

Either use a Maxima development version (Git) - or an older Gnuplot version.

Best regards, Wolfgang

**Comment by leafina2895** [comment]
Interestingly plotting seems to work outside of wxMaxima without any problems.

![image](https://github.com/user-attachments/assets/55886eb5-51e2-4d3f-9133-32914fc5b1e4)

It still sends the error message, but plots it in a different window anyways. (Still using Gnuplot 6 and Maxima 5.47.0)
Plotting in 2d works either way.

**Comment by johanwiden** [comment]
As a work-around one can tell wxMaxima not to use pm3d. Example:
wxplot3d( exp(-x^2 - y^2), [x,-2,2],[y,-2,2],[gnuplot_pm3d,false]);

**Comment by daute** [comment]
A new Maxima version was released on 2025-08-06, where the issue is solved.
And @johanwiden provided a workaround (thank you!).
(And it is not a wxMaxima problem...)
Closing the issue.

Best regards, Wolfgang
